src/main/scala/io/digitallibrary/bookapi/BookApiProperties.scala

```diff
@@ -71,7 +71,7 @@ object BookApiProperties extends LazyLogging {
   val LoginEndpoint = "https://digitallibrary.eu.auth0.com/authorize"
 
   val ImageApiHost = propOrElse("IMAGE_API_SERVER","image-api.gdl-local")
-  val InternalImageApiUrl = s"$ImageApiHost/image-api/v1/images"
+  val InternalImageApiUrl = s"$ImageApiHost/image-api/v2/images"
```

Comment:
@gunnarvelle Ok å gjøre denne endringen nå?

Reply:
Burde være uproblematisk. Urlen brukes for å hente ut et bilde fra image-api og begge versjonene oppfører seg likt.